botocore/config.py

```diff
@@ -235,6 +235,37 @@ class Config:
         specified service will be ignored.
 
         Defaults to None.
+
+    :type request_checksum_calculation: ``RequestChecksumCalculation``
```

Comment:
This seems wrong. The type is a string, we're using the enum to manage/represent that string. We shouldn't be adding additional configuration complexity to this if it can be avoided.

Reply:
Updated to `str`.